data/ecosystem/vendors.yaml

```diff
@@ -384,3 +384,10 @@
   contact: 'https://github.com/adnanrahic'
   oss: true
   commercial: true
+- name: Alibaba Cloud
+  distribution: true
+  nativeOTLP: true
+  url: 'https://www.alibabacloud.com/help/en/arms/tracing-analysis/get-started-with-tracing-analysis'
+  contact: 'opensource@alibaba-inc.com'
```

Comment:
update line1 of this file to contain opensource:

```
# cSpell:ignore Coralogix Cribl ITRS Kloud Logz Crowdstrike Humio Lumigo observ opensource Teletrace Uptrace Greptime KloudMate qryn ClickHouse Tracetest
```

Reply:
> update line1 of this file to contain opensource:
> 
> ```
> # cSpell:ignore Coralogix Cribl ITRS Kloud Logz Crowdstrike Humio Lumigo observ opensource Teletrace Uptrace Greptime KloudMate qryn ClickHouse Tracetest
> ```

ok, I added it.